spring-cloud-dataflow-common/spring-cloud-dataflow-common-test-docker/src/test/java/org/springframework/cloud/dataflow/common/test/docker/compose/configuration/AdditionalEnvironmentValidatorTests.java

```diff
@@ -15,30 +15,26 @@
  */
 package org.springframework.cloud.dataflow.common.test.docker.compose.configuration;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 import java.util.HashMap;
 import java.util.Map;
-import org.junit.Rule;
+
 import org.junit.Test;
```

Comment:
Since you are in here already, do you mind replacing Junit4 test w/ Junit5 test?